GOVERNANCE.md

```diff
@@ -47,28 +86,51 @@ Summaries of key decisions and discussions will be shared with the community to
 
 - **Concerns and Actions**: Members can voice concerns or propose actions by emailing `council@mailing-list` (TODO: or initiating a GitHub discussion, etc.).
 
-### Composition and Membership
+### 2.2.1 Composition and Membership
 
-The Council has a minimum of 3 elected members: 1 [Council Chairperson](#chairperson-of-the-council) and at least 2 members-at-large.
+The Council is composed of elected members who are recognized for their individual contributions and expertise in the open-source community. While members may be affiliated with various organizations, they are expected to prioritize the interests of the CF community in their decisions and actions.
 
-At least one-third of Council members are direct representatives from our projects, maintaining a strong voice for project needs and perspectives. To avoid any single entity's undue influence, no more than one Council member may represent any given company or legal entity.
+Council members are required to disclose any potential conflicts of interest, especially those related to their employment. This disclosure ensures transparency and helps maintain the integrity of the Council's decisions.
```

Comment:
Where is this disclosed? Is it part of the application to be a council member?

Is there an intention to publish bios of the council members? This would allow members of the community to see the possible conflicts of interest for the members

Reply:
Resolving this comment, but I've stashed this idea for later. Will come back to it.